examples/nemo/scripts/omp_cpu_trans.py

```diff
@@ -111,5 +111,6 @@ def trans(psyir):
                     loop_directive_trans=omp_loop_trans,
                     collapse=False,
                     privatise_arrays=(not NEMOV4 and
-                                      psyir.name not in PRIVATISATION_ISSUES)
+                                      psyir.name not in PRIVATISATION_ISSUES),
+                    enable_reductions=True
```

Comment:
@mn416 You could move this to the omp_gpu_trans.py in the same folder, in that script we have a flag to enable/disable reproducibility:
https://github.com/stfc/PSyclone/blob/5614b304dec4cdfe332cc53652473f24ab12b8ac/examples/nemo/scripts/omp_gpu_trans.py#L58-L59

so this would be:
`enable_reductions=not REPRODUCIBLE`

During the integration test this is build with both REPRODUCIBLE set to true and false.

@LonelyCat124 For correctness I would suggest manually doing
./makenemo ....
and
REPRODUCIBLE=1 ./makenemo ....

and diff the generated code to see if all the changes make sense

(another option is to make a longer run and see if the run.stat values don't diverge above a tolerance, but we can do this later)

Reply:
These are part of inegration tests. Aidan will run these for you as part of the review process.